src/components/network/network-table.js

```diff
@@ -85,6 +87,10 @@ const useStyles = makeStyles((theme) => ({
         marginTop: theme.spacing(2),
         marginLeft: theme.spacing(6),
     },
+    invalidNode: {
+        marginTop: '8px',
+        marginLeft: '50px',
```

Comment:
Use theme.spacing(X) instead of pixels ?
theme.spacing(1) = 8px;
theme.spacing(6.25) = 50px;

Reply:
Done